### PR TITLE
Add connect and sync operations to $morphToMany relation during model creation

### PR DESCRIPTION
**Pull request by megawubs**
- [x] Added or updated tests
- [x] Added Docs for all relevant versions
- [x] Updated the changelog

**Related Issue/Intent**
During model creation I want to be able to attach or sync existing models to an morphToMany relationship.

**Changes**
Added checks for `connect` and `sync` in `executeCreateMorphToMany`

**Breaking changes**

### CODE REVIEW

**Comment by megawubs** [comment]
I'll add tests and documentation when I have the time. For now, this is the basics

**Comment by spawnia** [comment]
@megawubs The implementation looks fine, i can merge this when you added docs/tests/changelog.

**Comment by megawubs** [comment]
@spawnia I added the tests and updated the docs.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+ - Support for syncing and connecting existing models in morphToMany relationships
```
Comment (spawnia) [inline CHANGELOG.md]:
```suggestion
### Added
 - Sync and connect existing models in morphToMany relations using nested mutations https://github.com/nuwave/lighthouse/pull/707
```

File: tests/Integration/Execution/MutationExecutor/MorphToManyTest.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    /**
+     * @test
+     */
+    public function itCanCreateATaskWithExistingTagsByUsingConnect(){
+        $id = factory(Tag::class)->create(['name' => 'php'])->id;
+
+        $this->query('
+            mutation {
+            createTask(input: {
```
Comment (spawnia) [inline tests/Integration/Execution/MutationExecutor/MorphToManyTest.php]: This does not follow our standard formatting.

File: tests/Integration/Execution/MutationExecutor/MorphToManyTest.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                    tags: {
+                        sync: [1]
+                    }
+                }){
+                    tags{
+                            id
+                        }
+                    }
+                }
+            ')->assertJson([
```
Comment (spawnia) [inline tests/Integration/Execution/MutationExecutor/MorphToManyTest.php]: Indentation is off

File: tests/Integration/Execution/MutationExecutor/MorphToManyTest.php [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    }
+
+    /**
+     * @test
+     */
+    public function itCanCreateATaskWithExistingTagsByUsingSync(){
+        $id = factory(Tag::class)->create(['name' => 'php'])->id;
+
+        $this->query('
+            mutation {
```
Comment (spawnia) [inline tests/Integration/Execution/MutationExecutor/MorphToManyTest.php]: no indentation here

**Comment by spawnia** [comment]
@megawubs Looking better. The tests look solid.

I am still missing an actual documentation of this new feature, i think it should go at the bottom of the `relationships.md`, after `MorphTo`. Maybe you can add a more comprehensive example with a bit of description.

**Comment by megawubs** [comment]
@spawnia I updated the docs and added another test to see if `create` also works, as this was in the code but i could not find a test for it.

**Comment by spawnia** [comment]
Nice work @megawubs, i will review soon and get this merged!

**Pull request merged**